lib/cocoapods/installer.rb

```diff
@@ -210,11 +207,12 @@ def generate_pods_project(generator = create_generator)
         @target_installation_results = pod_project_generation_result.target_installation_results
         @pods_project = pod_project_generation_result.project
         run_podfile_post_install_hooks
-        generator.write(@pods_project, @target_installation_results)
+        Xcode::PodsProjectGenerator.write(@pods_project, target_installation_results, @sandbox.project_path, installation_options.deterministic_uuids?)
```

Comment:
follow up to https://github.com/CocoaPods/CocoaPods/pull/8221#discussion_r228325405